### PR TITLE
Money to wallet routing

### PR DESCRIPTION
**Pull request by RetroSabian**
Routed
Added Credit card payment, and success page
Reducers for credit card details
Top up API call in reducer

### CODE REVIEW

File: src/Components/MoneyToEWallet/CompletePaymentCreditComponent.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+class CompletePaymentCreditComponent extends React.PureComponent {
+    constructor(props) {
+        super(props);
+    }
+
+    async uploadWallet(){
+        const uuidv4 = require('uuid/v4');
+        const userID = { userId: '1' };
+        const jwt = 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJqYWNrc29hcCIsInVpZCI6MSwiY2giOiJZclJWSUdMbTVRIiwicm9sZXMiOiIiLCJpc3MiOiJodHRwOlwvXC9qaW5pLmd1cnUiLCJleHAiOjE1NTQ1MzkyMDZ9.nIadvzD_EsPI-toowhqfjjU6nczmXOFO5aFpUBdsw0k';
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/CompletePaymentCreditComponent.js]: replace with logged in user JWT once login is completed

File: src/Components/MoneyToEWallet/CompletePaymentCreditComponent.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                placeholder = {'cvv'}
+                onChangeText = { value => {
+                    this.props.loadActions.setCardCVV(value);
+                }}
+            />
+            <Button
+                title = "CONTINUE"
+                type = "Solid Button"
+                color = "yellow"
+                onPress= { () => this.uploadWallet()}
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/CompletePaymentCreditComponent.js]: can be this.uploadWallet

File: src/Components/MoneyToEWallet/PaymentOptionComponent.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { View , Text , Button } from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+class PaymentOptionComponent extends React.PureComponent {
+    constructor(props) {
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/PaymentOptionComponent.js]: no constructor required

File: src/Components/MoneyToEWallet/PaymentOptionComponent.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    navigation: PropTypes.shape ({
+        navigate: PropTypes.func
+    }).isRequired
+};
+
+const mapStateToProps = (state) => ({
+    walletUploadState: state.walletUploadState
+});
+
+export default connect(mapStateToProps, null)(PaymentOptionComponent);
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/PaymentOptionComponent.js]: dont need to pass the null in the place of mapActions to props. simply pass nothing

File: src/Components/MoneyToEWallet/SuccessComponent.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    render() {
+        return <View>
+            <Text>Successful!</Text>
+            <Text>R {this.props.walletUploadState.amountToAdd}</Text>
+            <Text>Added to your wallet</Text>
+            <Text>Updated Balance</Text>
+            <Text>R {this.props.walletUploadState.walletBalanceState.balance}</Text>
+            <Button
+                title = "+ Add more money"
+                onPress = { () => this.loadMore()}
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/SuccessComponent.js]: can be this.loadMore

File: src/Components/MoneyToEWallet/CompletePaymentCreditComponent.js (hunk 2 above)
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/CompletePaymentCreditComponent.js]: done

File: src/Components/MoneyToEWallet/PaymentOptionComponent.js (hunk 4 above)
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/PaymentOptionComponent.js]: done

File: src/Components/Home/SubViews/Introduction.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    },
+    backgroundImage: {
+        flex: 1,
+        width: '100%',
+        height: '100%',
+        resizeMode: 'cover'
+    },
+});
+
+class Introduction extends React.PureComponent {
```
Comment (GeromeSchutte) [inline src/Components/Home/SubViews/Introduction.js]: component and file name doesn't follow component naming convention

Comment (RetroSabian) [reply src/Components/Home/SubViews/Introduction.js]: removed

**Pull request merged**